en/admin/installation/installation.md

```diff
@@ -9,6 +9,12 @@ git clone https://github.com/wallabag/wallabag.git
 cd wallabag && make install
 ```
 
+If you want to use LDAP, replace last command with
+```bash
+cd wallabag && make install LDAP_ENABLED=true
+```
```

Comment:
@nicosomb : and false == absent, you don’t need to specify features that you don’t want :p